pipeline/translate/collect.sh

```diff
@@ -1,25 +1,53 @@
 #!/bin/bash
 #
-# Merges translation outputs into a dataset
+# Collects chunked translation data of the form "out-file.N.out" where N is a number.
+# The datasets are initially chunked so that tasks can work on smaller sets of data
```

Comment:
suggestion (non-blocking): Perhaps something like "The datasets are chunked in earlier in the pipeline by `split-{corpus,mono}.sh`'? (The current usage of "initially" here made it sound at first like the chunking was happening _here_.